cfn-resources/auditing/cmd/resource/resource.go

```diff
@@ -27,10 +28,10 @@ import (
 	log "github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progress_events "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
-	mongodbatlas "go.mongodb.org/atlas/mongodbatlas"
+	atlasSDK "go.mongodb.org/atlas-sdk/v20230201008/admin"
```

Comment:
knitpit: I prefer not use an alias here and just refer to admin.Methods..

Reply:
I don't find `admin` to be very readable here so I'd prefer to use the alias (do let me know in case there's a better one we could use)

Reply:
updated